classes/class-wc-connect-jetpack.php

```diff
@@ -152,5 +152,43 @@ public static function tracks_record_event( $user, $event_type, $data ) {
 			}
 			return false;
 		}
+
+		/**
+		 * Determines if the current user is connected to Jetpack
+		 * @return bool Whether or nor the current user is connected to Jetpack
+		 */
+		public static function is_current_user_connected() {
+			if ( defined( JETPACK_MASTER_USER ) ) {
+				$user_token = WC_Connect_Jetpack::get_master_user_access_token( JETPACK_MASTER_USER );
+
+				return ( isset( $user_token->external_user_id ) && get_current_user_id() === $user_token->external_user_id );
+			} else {
```

Comment:
No need for `else` here, since in the previous `if` there's already an early return.

I think that for legibility we can just do:

```
if ( defined( JETPACK_MASTER_USER ) ) {
  ...
  return ...;
}

if ( class_exists( '\Automattic\Jetpack\Connection\Manager' ) && ... ) {
  ...
  return ...;
}

return false;
```

Also - should we put the newer code (`class_exists( '\Automattic\Jetpack\Connection\Manager' )`) above the check for the constant, so that the newer code is returned earlier?

Thoughts?

Reply:
Sounds good @frosso! Those are some nice suggestions. I just pushed a new commit.